xml/System.Text.RegularExpressions/GroupCollection.xml

```diff
@@ -772,7 +788,15 @@
       <Parameters />
       <Docs>
         <summary>Clears the collection. Calling this method always throws <see cref="T:System.NotSupportedException" />.</summary>
```

Comment:
@pgovind This is a good example of how all `summaries` and `returns` in non-supported APIs should look:
```suggestion
        <summary>Calling this method always throws <see cref="T:System.NotSupportedException" />.</summary>
```